### PR TITLE
Pretty print inexact

### PR DESCRIPTION
**Pull request by jbrown215**
Adds pretty printing for the new inexact object type flow syntax as of 0.84.0. Depends on https://github.com/prettier/prettier/pull/5303

- [x] I’ve added tests to confirm my change works.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: tests/es6modules/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default async function foo() {}
 exports[`export_default_function_declaration_async.js - flow-verify 1`] = `
 export default async function foo() {}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-export default (async function foo() {});
+export default async function foo() {}
```
Comment (lydell) [inline tests/es6modules/__snapshots__/jsfmt.spec.js.snap]: See https://github.com/prettier/prettier/pull/5303#discussion_r228407664

Comment (jbrown215) [reply tests/es6modules/__snapshots__/jsfmt.spec.js.snap]: resolved in #5303

File: tests/flow_object_inexact/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+type V = {x: {...}, y: {x: {...}, a: number, b: number, c: number, d: number, e: number, f: number, ...}, z: {...}, foo: number, bar: {foo: number, ...}, ...};
+
+function test(x: {foo: number, bar: number, baz: number, qux: nunber, a: number, b: number, c: {a: number, ...}, ...}) { return x; }
+function test(x: {foo: number, bar: number, baz: number, qux: nunber, a: number, b: number, c: {a: number, b: number, c: number, d: number, e: number, f: number, g: number, h: number, i: number, ...}, ...}) { return x; }
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+//@flow
+type T = {
+  a: number,
+  ...,
```
Comment (j-f1) [inline tests/flow_object_inexact/__snapshots__/jsfmt.spec.js.snap]: Should this have a trailing comma?

**Comment by jbrown215** [comment]
Thanks for the quick reviews!

**Comment by j-f1** [comment]
Looks like the lock file got changed. Are you on the latest Yarn @jbrown215?

**Comment by jbrown215** [comment]
That was from the change this depended on-- rebased onto master.

**Pull request merged**